src/inc/msquic.h

```diff
@@ -1714,6 +1725,9 @@ typedef struct QUIC_API_TABLE {
     QUIC_CONNECTION_COMP_RESUMPTION_FN  ConnectionResumptionTicketValidationComplete; // Available from v2.2
     QUIC_CONNECTION_COMP_CERT_FN        ConnectionCertificateValidationComplete;      // Available from v2.2
 
+    QUIC_CONNECTION_OPEN_IN_PARTITION_FN
```

Comment:
This is added in the middle of the struct? What happens if an old client gets the API table?

Reply:
It's only before preview stuff. Nothing below has shipped in a release.